packages/ListBox/src/helpers/handleKeyboardKeys.js

```diff
@@ -4,17 +4,13 @@ import useListBox from "../useListBox";
 export const handleKeyDownKeyboardKeys = ({ providedProps, state, dispatch, onChangeContext }) => event => {
   if (providedProps.isDisabled) return;
 
-  switch (event.key) {
-    case "ArrowUp":
-      handleArrowKeys({ event, providedProps, state, dispatch, onChangeContext, isArrowDown: false });
-      break;
-
-    case "ArrowDown":
-      handleArrowKeys({ event, providedProps, state, dispatch, onChangeContext, isArrowDown: true });
-      break;
+  if (event.key === " ") {
+    // prevents spacebar to scroll the page
+    event.preventDefault();
+  }
 
-    default:
-      break;
+  if (["ArrowUp", "ArrowDown"].includes(event.key)) {
+    handleArrowKeys({ event, state, dispatch, onChangeContext });
```

Comment:
Maybe this is the place to add `event.preventDefault();` so that the page doesn't scroll?

Reply:
was in the trigger not in here I added it.